addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -638,11 +638,9 @@ options.Class.include({
      */
     _makeSCSSCusto: async function (url, values, defaultValue = 'null') {
         return this._rpc({
-            route: '/website/make_scss_custo',
-            params: {
-                'url': url,
-                'values': _.mapObject(values, v => v || defaultValue),
-            },
+            model: 'web_editor.assets',
+            method: 'make_scss_customization',
+            args: [[], url, _.mapObject(values, v => v || defaultValue)],
```

Comment:
```suggestion
            args: [url, _.mapObject(values, v => v || defaultValue)],
```
And add @api.model on `make_scss_customization`, `args: [[]` is always a warning that `@api.model` is missing
There is probably a lot of those missing actually in those editor related methods